solidity/contracts/bridge/Bridge.sol

```diff
@@ -144,6 +165,14 @@ contract Bridge is Ownable {
     ///         prefix refers to the transaction which created that main UTXO.
     mapping(bytes20 => bytes32) public mainUtxos;
 
+    // TODO: Documentation.
+    // Key: keccak256(walletPubKeyHash | redeemerOutputHash)
+    mapping(uint256 => RedemptionRequest) public redemptionRequests;
```

Comment:
What do you think about naming it `pendingRedemptionRequests`? My understanding is that we will store failed redemption requests in the second mapping forever. If we name this field just `redemptionRequests`, this could suggest *all* redemptions ever requested are stored here.

Reply:
Yeah, we could rename it that way. Also, note that the fact we store keys in `redemptionFaults` forever means that a faulty key (so wallet and output hash pair) cannot be reused for future requests. This is obviously a UX downside but it simplifies a lot and reduces the attack surface. Of course, keys used for successful request (the majority) can be normally reused in future as described in https://github.com/keep-network/tbtc-v2/pull/132

Reply:
See https://github.com/keep-network/tbtc-v2/pull/128/commits/022ad3feafa9295a3cebcc2f65187ea0d628251b